spec/models/concerns/json_schema_validator_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+
+RSpec.describe JsonSchemaValidator, type: :validator do
+  schema = {
+    'name' => { required: true, type: 'string' },
+    'age' => { required: true, type: 'integer' },
+    'is_active' => { required: false, type: 'boolean' },
+    'tags' => { required: false, type: 'array' },
+    'address' => {
+      required: false,
+      type: 'hash',
+      properties: {
+        'street' => { required: true, type: 'string' },
+        'city' => { required: true, type: 'string' }
+      }
+    }
+  }.freeze
+  # Create a simple test model for validation
+  before_all do
+    # rubocop:disable Lint/ConstantDefinitionInBlock
+    # rubocop:disable RSpec/LeakyConstantDeclaration
+    TestModelForJSONValidation = Struct.new(:additional_attributes) do
+      include ActiveModel::Validations
+
+      validates_with JsonSchemaValidator, schema: schema
+    end
+    # rubocop:enable Lint/ConstantDefinitionInBlock
+    # rubocop:enable RSpec/LeakyConstantDeclaration
+  end
```

Comment:
Couldn't solve these Rubocop warnings, would need help if we have to solve this 🥲

Reply:
let's go ahead with this for now. I will be doing an audit for all the disabled cops during the rails upgrade. will have a look at this then. 
ref: https://linear.app/chatwoot/issue/CW-1574/renable-the-disabled-rubocop-rules